test.js

```diff
@@ -0,0 +1,37 @@
+const assert = require('assert')
+const { describe, it } = require('mocha')
+const parser = require('./')
+
+describe('parser', () => {
+  describe('<type> ":" <text>', () => {
+    it('parses summary with no scope', () => {
+      const message = parser('fix: a really weird bug')
+      assert.strictEqual(message.type, 'message')
+      const summary = message.children[0]
+      assert.strictEqual(summary.type, 'summary')
+      const type = summary.children[0]
+      assert.strictEqual(type.type, 'type')
+      assert.strictEqual(type.value, 'fix')
```

Comment:
should we consider using snapshot testing, or is it better to be explicit about the nodes?

Reply:
Those snapshots won't change that often I think. Either way looks fine to me.